README.md

```diff
@@ -146,30 +59,19 @@ The native Android Studio and XCode projects have a build hook which will find t
 - `template-files` contains templates for files that require private keys. They are populated using the keys in `template-files/keys.json`.
 - `template-files/ios/dependencies.json` specifies the CocoaPods dependencies of the app.
 
-## Tests
-
-### iOS
-
-For native XCTest unit tests:
-
-- Press Command+U in XCode to build and test the `Tests` unit test target.
-- Alternatively, run `fastlane ios test` from the parent directory of `ios`.
-
-For JS integration tests, test the `ExponentIntegrationTests` target (not included in the default test scheme). This target requires you to configure `EXTestEnvironment.plist` with a key `testSuiteUrl` whose value is the URL to load some version of Expo's [test-suite](apps/test-suite) app. This will run a bunch of Jasmine tests against the Expo SDK.
+## 👏 Contributing
 
-## Contributing
+If you like Expo and want to help make it better then check out our [contributing guide](/CONTRIBUTING.md)!
```

Comment:
not sure where to put it exactly but we should mention that people should go to http://github.com/expo/expo-cli to contribute to expo-cli